src/scripts/check_distance.py

```
@@ -0,0 +1,113 @@
+    # 1. rag_service와 동일한 모델 로드 함수 사용
+    embedding_model, _ = get_models()
+
+    # 2. DB 엔진 생성
+    engine = create_async_engine(settings.DATABASE_URL, echo=False)
+
+    # 3. 질문 임베딩 (rag_service와 동일한 방식)
+    try:
+        text_input = TextEmbeddingInput(text=TEST_QUERY, task_type="RETRIEVAL_QUERY")
+        embeddings = embedding_model.get_embeddings([text_input])
+        query_vector = embeddings[0].values
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)